libs/s3/s3_multipart_test_lib.py

```diff
@@ -225,10 +227,12 @@ def upload_precalculated_parts(self,
                         data, bucket_name, object_name, upload_id=upload_id,
                         part_number=int(partnum) + 1)
                     LOGGER.debug("Part : %s", str(part))
-                    uploaded_parts.append({"PartNumber": int(partnum) + 1, "ETag": part["ETag"]})
+                    uploaded_parts.append({"PartNumber": int(partnum) + 1,
+                                           "ETag": part["ETag"]}) # nosec
                     md5_digests[int(partnum)] = md5(data).digest()
```

Comment:
It looks like the nosec comment needs to be added to line 232

Reply:
sure